linode/manager
# feat: [M3-7257] - Invalidate VPC-related queries after deleting a linode

## Description 📝
- After deleting a linode, because we didn't invalidate any VPC related queries, the VPC pages would have incorrect information unless we refreshed the page. This change invalidates VPC related queries when deleting a linode so that the vpc pages are up to date.

## Changes  🔄
List any change relevant to the reviewer.
- Added utility function to get all vpcs a linode is assigned to with, based on its configs
- Invalidate vpc-related queries for vpcs a linode is assigned to 
- Added some extra spacing underneath the 'Reboot linode banner' on a VPC details page -- seems like there was spacing missing, added more to make it align more with the mockups 
- updated VPC todo comments to be consistent - `//@TODO VPC: ...todos...`

## Preview 📷
Before:
https://github.com/linode/manager/assets/139280159/eb4fae00-dbf0-49af-b54f-bb3ec449e307

After:
https://github.com/linode/manager/assets/139280159/24012745-b602-4aef-abd6-abe724c85c3c

## How to test 🧪

### Prerequisites
(How to setup test environment)
Use the dev environment, with VPC tags, 
- Have a linode and at least two vpc's created (each with at least one subnet)

### Verification steps 
(How to verify changes)
Verify that the VPC landing page and specific VPC detail pages are correctly updated after you delete a linode that is assigned to a VPC. Test the following cases:
- Linode assigned to one vpc one subnet
- Linode assigned to one vpc (multiple subnets within that VPC)
- Linode assigned to two+ VPCs

### Tests
- `yarn test LinodesLanding/utils.test.ts`

## As an Author I have considered 🤔

*Check all that apply*
